app/services/db-export/send-to-s3-bucket.service.js

```diff
@@ -1,79 +1,55 @@
 'use strict'
 
 /**
- * Uploads a folders worth of files to our S3 bucket
+ * Sends a file to our AWS S3 bucket
  * @module SendToS3BucketService
  */
 
 const fsPromises = require('fs').promises
 const path = require('path')
 const { PutObjectCommand, S3Client } = require('@aws-sdk/client-s3')
```

Comment:
Eeek! I can see you removed the line but actually, our [convention](https://medium.com/selleo/essential-rubyonrails-patterns-part-1-service-objects-1af9f9573ca1) _is_ to put a break before the config.